payloads/extensions/linux_mount.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# LINUX_MOUNT v1 by @drapl0n
+# Auto mounts BashBunny on GNU/Linux systems.
+# NOTE: Mount path is stored in variable "lmnt".
+# Usage: LINUX_MOUNT - to automatically mount BashBunny.
+#	 LINUX_UMOUNT - to unmount mounted BashBunny.
+
+function LINUX_MOUNT() {
+	Q DELAY 1000
+	Q CTRL-ALT t
+	Q DELAY 1000
+	Q STRING unset HISTFILE
+	Q ENTER
+	Q DELAY 1000
+	Q STRING disk='$(lsblk -fs | grep BashBunny | awk '\'{print\ '$1'}\'\)''
+	Q ENTER
+	Q DELAY 200
+	Q STRING udisksctl mount -b /dev/'$disk'
+	Q ENTER
+	Q DELAY 2000
+	Q STRING lmnt='$(lsblk | grep $disk | awk '\'{print\ '$7'}\'\)''
+	Q ENTER
+	Q DELAY 500
+}
+function LINUX_UMOUNT() {
+	Q DELAY 1000
+	Q STRING udisksctl unmount -b /dev/'$disk'
```

Comment:
Why is there a DELAY at the beginning of th is function too?

Reply:
payload between LINUX_MOUNT and LINUX_UMOUNT function may not be ended with a delay by author, in order to mitigate that delay has been added at beginning.

Reply:
That should be handled in the payload